### PR TITLE
Feat: Integrate Project Detail Info page with mock-server

### PR DESCRIPTION
**Pull request by rkpattnaik780**
### Type of change

- Enhancement
- Bugfix

### Description

* Project detail info page integrated with mock.
* JSON view of Device Registration management fixed.
* Enable status implemented.
* Delete project added to ProjectDetailHeader.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Resolve all warnings
- [ ] Update license
- [ ] Write test cases for null checks in Unit Tests
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: console/console-init/ui/mock-console-server/mock-console-server.js [hunk 1]
```diff
@@ -2568,7 +2574,11 @@ const resolvers = {
     disableIotProjects: (parent, args) => {
       runOperationForAll(args.input, t => disableIotProject(t));
       return true;
-    }
+    },
+    enableIotProjects: (parent, args) => {
+      runOperationForAll(args.input, t => enableIotProject(t));
+      return true;
+    },
```
Comment (jbtrystram) [inline console/console-init/ui/mock-console-server/mock-console-server.js:2581]: let's regroup this two queries in one

File: console/console-init/ui/mock-console-server/schema.js [hunk 2]
```diff
@@ -675,6 +675,7 @@ const typeDefs = gql`
     ): Boolean
     deleteIotProjects(input: [ObjectMeta_v1_Input!]!): Boolean
     disableIotProjects(input: [ObjectMeta_v1_Input!]!): Boolean
+    enableIotProjects(input: [ObjectMeta_v1_Input!]!): Boolean
```
Comment (jbtrystram) [inline console/console-init/ui/mock-console-server/schema.js:678]: how about 
```
toggleIotProjects(input: [ObjectMeta_v1_Input!]!, state Boolean!): Boolean
```

File: console/console-init/ui/mock-console-server/mock-console-server.js (hunk 1 above)
Comment (rkpattnaik780) [reply console/console-init/ui/mock-console-server/mock-console-server.js:2581]: Will address it in my next PR.

File: console/console-init/ui/mock-console-server/schema.js (hunk 2 above)
Comment (rkpattnaik780) [reply console/console-init/ui/mock-console-server/schema.js:678]: Will address it in my next PR.

**Pull request merged**